### ISSUE TITLE
New Weapon Modifiers?

### ISSUE THREAD

**Issue by BardBlue** [Lknechtli/BetterCombatEvolved#9]
Weapon modifiers seem pretty limited overall. I'm especially sad that none of them gives bonuses for alchemy (as far as I know), like increased max toxicity, or longer potion length. But I'm not sure if it's possible to add new modifiers for weapons? Probably the same case with armor, but not quite sure yet.

Even if there aren't ways to add brand new modifiers, I think you should take a look at the Relic Weapons Redone mod and maybe apply some of the things he's done. Vanilla relic weapons are mostly boring and weak. I do think he might've gone overboard on some weapons (for example, 50% crit rate on one sword), but not too much. Most of the time you'll be upgrading a sword based mostly on its damage anyway, but it'd be nice if their bonuses were good enough to consider taking a damage hit for. For example, a +1000 health sword with +25 sign intensity, but relatively low damage for its level.

Or you could even give some swords maluses, like -1000 health for a pretty high damage sword for the level. And always nice when lore appropriate, but that definitely takes a long time to figure out.

Anyway, is this possibly on your agenda?

**Comment by Lknechtli** [comment]
I think that figuring out skill balance is our highest priority right now.  After we figure that out, we'll start looking at other ways to make equipment choices more varied

**Comment by BardBlue** [comment]
Alright, makes sense. I do think the loot scene, for what feels like a pretty loot-driven game, is rather subpar. More varied items would do a lot for the game, just as better trophies add a lot.

**Comment by Lknechtli** [comment]
I agree about that, but it's not something that could be done any way other
than completely manually.  There's only so much you can do with randomly
generated stats to make things interesting (see borderlands and its
trillion guns you won't use).

If we do work decide to do something along the lines of Relic Weapons
Redone, it will probably be the primary effort of that release.

I'm currently using Relic Weapons redone right now myself :)
